sources/platform/integrations/llama.md

```diff
@@ -26,36 +26,58 @@ This example focuses on integrating Apify with LlamaIndex using the Apify Actor.
 
 Before we start with the integration, we need to install all dependencies:
 
-`pip install apify-client llama-index`
+`pip install llama-index-readers-apify`
```

Comment:
You need to install `llama-index-core` as well.

Reply:
It is actually not required as `llama-index-core` is a dependency of `llama-index-readers-apify`. However, on second thought, it might be better to install it explicitly along with `apify-client`.